lucille-core/src/test/resources/ParseFilePathTest/config.conf

```diff
@@ -0,0 +1,4 @@
+{
+  class = "com.kmwllc.lucille.stage.ParseFilePath"
+  file_path_field = "file_path"

```

Comment:
need coverage for the other parameters: file_step, uppercase_extension, include_heirarchy